ons_alpha/core/blocks/stream_blocks.py

```diff
@@ -13,8 +14,22 @@
     PanelBlock,
     RelatedContentBlock,
     RelatedLinksBlock,
-    TableBlock,
 )
+from ons_alpha.core.blocks import (
+    TableBlock as OldTableBlock,
+)
+
+
+class ONSTableBlock(StructBlock):
+    heading = CharBlock(required=True, help_text="Add a heading for the table.")
+    table = TableBlock(required=True, help_text="Add the table data here.")
+    source = CharBlock(required=False, help_text="Add the source of the table data if applicable.")
+    footnotes = RichTextBlock(features=["bold", "italic"], required=False, help_text="Add any footnotes for the table.")
+
+    class Meta:
```

Comment:
I see that the table child block is not rendering properly. Use

```suggestion

    def get_context(self, value, parent_context=None):
        context = super().get_context(value, parent_context=parent_context)
        context["table"] = self.child_blocks["table"].render(value["table"])
        return context


    class Meta:
```

and the simply `{{ table }}` in `ons_table_block.html`